BowlingBall.Tests/GameFixture.cs

```diff
@@ -5,10 +5,62 @@ namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+        private Game game;
+
+        public GameFixture()
+        {
+            game = new Game();
+        }
+
+        private void RollMany(int n, int pins)
+        {

```

Comment:
Fixture file should contain only [Fact], so this logic should not be in this class